Makefile

```diff
@@ -36,7 +36,7 @@ operator-sdk: get-dep
 	fi
 
 imagebuilder:
-	@if ! type -p imagebuilder ; \
+	@if [ $${USE_IMAGE_STREAM:-false} = false ] && ! type -p imagebuilder ; \
```

Comment:
why is it `$$`?

Reply:
That's a Makefile-ism - one `$` will attempt to evaluate to a make variable and will fail if one is not defined - `$$` escapes them so that it will evaluate to one `$` in the shell context